pkg/logs/logs.go

```diff
@@ -0,0 +1,39 @@
+package mpclogs
+
+import (
+	"flag"
+	zap2 "go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+const DebugLevel = 1
+
+func InitLogger(logLevel, stackTraceLevel string) error {
```

Comment:
I don't think we need any of this - to log at a higer (less siginificant) level, all we need to do is `log.V(1).Info(...)` with a normal logr object.

Reply:
Sorry I didn't get the message. This code allows to start the controller with any desired logging level. I.e. by default it will be `info` (so no debugs), but using `--zap-log-level=debug` (or any other valig level) it can be switched on.

Reply:
I think the built-in zap options aught to be enough. Also I don't think this aught to be very configurable. I don't want to be unable to debug a production issue because the messages I need happen to be in a level we were not tracking. 

In other words, IMO level filtering should be done when querying at the destination (e.g. Splunk) and not at the source.

Reply:
I would say it is not any easier to debug the issue when log is overpolluted by meaningless messages like `pipelinerun xxxx was probably deleted`  et cetera. They doesnt carry any payload, just the log size grows enormous. The idea was to mute such unifromative stuff by def.

Reply:
if you really think a message is unnecessary, remove it from the code, otherwise you can never know what you'll need to debug stuff for users.

Reply:
OK, so what should we leave in the PR then? To me it is have not much meaning move the log levels without filtering ability.